pkg/certificates/certificatesclient/client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package certificatesclient
```

Comment:
moved from `certificates` package

Reply:
I would call this package just `client` the problem that I see is that if you need to import several clients then you have to use package aliases.

Do we always use `{parent package name}client`?

Reply:
I'm not sure.